frontend/src/components/CalendarQalendar.vue

```diff
@@ -312,19 +315,17 @@ const config = ref({
     length: timeSlotDuration.value, // Accepts [15, 30, 60]
     height: timeSlotHeight.value, // pixel height of each length
   },
-  dayBoundaries: dayBoundary,
+  dayBoundaries: {
+    start: dayBoundary.value.start,
+    end: dayBoundary.value.end
+  },
```

Comment:
At the point of initializing this config object, dayBoundary.value holds the default values 0, 24 making Qalendar span the container element over that range. Updating the boundary later in the watcher causes Qalendar to recalculate the displayed time rows, but it doesn't update the row height.